psst-gui/src/ui/album.rs

```diff
@@ -99,36 +99,37 @@ fn rounded_cover_widget(size: f64) -> impl Widget<Arc<Album>> {
     cover_widget(size).clip(Size::new(size, size).to_rounded_rect(4.0))
 }
 
-pub fn album_widget() -> impl Widget<WithCtx<Arc<Album>>> {
-    let album_cover = rounded_cover_widget(theme::grid(6.0));
-
-    let album_name = Flex::row()
-        .with_child(
-            Label::raw()
-                .with_font(theme::UI_FONT_MEDIUM)
-                .with_line_break_mode(LineBreaking::Clip)
-                .lens(Album::name.in_arc()),
-        )
-        .with_spacer(theme::grid(0.5))
-        .with_child(ViewSwitcher::new(
-            |album: &Arc<Album>, _| album.has_explicit(),
-            |selector: &bool, _, _| match selector {
-                true => icons::EXPLICIT.scale(theme::ICON_SIZE_TINY).boxed(),
-                false => Box::new(Flex::column()),
-            },
-        ));
+pub fn album_widget(horizontal: bool) -> impl Widget<WithCtx<Arc<Album>>> {
+    let album_cover = if horizontal { rounded_cover_widget(theme::grid(16.0)) } else { rounded_cover_widget(theme::grid(6.0)) };
+
+    let album_name = if horizontal { Flex::column() } else { Flex::row() }
```

Comment:
If you're doing a lot of these if/else horizontal it might be better to put them all into a single clause?

Reply:
Yes very true!